parlai/tasks/self_chat/worlds.py

```diff
@@ -124,16 +123,11 @@ def parley(self):
 
         if self.turn_cnt == 0:
             self.acts = [None, None]
-            # choose speaking order:
-            if random.choice([0, 1]):
```

Comment:
so is the order always the same now?

Reply:
yeah. in self chat they are the exact same agent. so it shouldn't matter at all. the personas are chosen randomly.